srcpkgs/python-xmldiff/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'xmldiff'
+pkgname=python-xmldiff
+version=1.1.1
+revision=1
+noarch=yes
+wrksrc="xmldiff-${version}"
+build_style=python-module
+pycompile_module="xmldiff"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
+depends="python-future python-lxml python-six"
+short_desc="Python2 classes to diff XML files"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
```

Comment:
If you're introducing a new package take maintainership. Plus Juan RP isn't present over a year